CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 ################################################################
 # general configuration
 ################################################################
-cmake_minimum_required (VERSION 3.12.0)
+cmake_minimum_required (VERSION 3.14.0)
 project(Vampire)
 
+include(CheckIPOSupported)
+include(CheckPIESupported)
+
```

Comment:
Just lifted the "include section" to the top. These are CMake-supplied optional modules that we need later.